---
subscriber_topic: /dock
publisher_topic: /dock_status
robot_availability_topic: /robot_description
idle_sleep_duration: 1.0
queue_size: 10